.github/workflows/build.yaml

```diff
@@ -38,12 +38,20 @@ jobs:
           distribution: 'zulu'
           java-version: 11
 
-      # Clang is seemingly already installed on Ubuntu but fails to build the project with errors
-      # like "file not found" for `#include <string>`. Explicitly (re?)installing fixes things.
-      - run: sudo apt-get install clang
+      - name: Build native library (Linux)
         if: matrix.os == 'ubuntu-latest'
-
-      - name: Build native library
+        run: |
+          cp -a . ../zipline-dockerbuild
+          docker run \
+            --mount src=`pwd`/../zipline-dockerbuild,target=/zipline,type=bind \
```

Comment:
This syntax is awful. I'll use the old volume syntax until they tear it from my cold, dead hands.

Reply:
Fixed.

Reply:
I mean you didn't have to change it. This is what they recommend now. It's just so yucky.